### DISCUSSION TITLE
Can't get lights to automate anymore

### DISCUSSION THREAD

**Discussion by ReplicaObscura** [jseidl/magic-areas#240] [Q&A] [answered]
Magic Areas stopped working for me a couple of weeks ago, seemingly at random when I rebooted. Since then, I tried deleting all my areas and rebooting, then resetting them up, but the light automations still don't do anything.

Version 3.0 has come out since then, and I did update, and then delete, reboot, and re-set up all my areas again, but it still doesn't work. Ditto for the 3.0.1 update.

I have motion sensors and lights in the areas, I'm not including or excluding any entities, I tried it without enabling any other optional features because I thought maybe one of those was causing an issue...

I tried enabling Info logging for magic_areas, and all it seems to indicate is that it starts setting up one of my areas, and then there are no further logs. I tried to enable debug logging, but there are too many logs for me to even make sense of.

I am not sure where to turn but I really miss my light automations.

**Comment by jseidl** [comment, marked as answer]
3.0 introduced a switch that controls whether the light automations are enabled. The naming patten is "Area Light Control ($area)". Did you turn those one? It's disabled by default.

**Reply by ReplicaObscura** [reply]
I did not realize turning that switch on was needed now, I bet that's it. Will mark your response the answer as soon as I test

**Reply by ReplicaObscura** [reply]
That was totally it, it's working great now. Thanks so much!
